contrib/IBM/sarama/sarama.go

```diff
@@ -0,0 +1,304 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+// Package sarama provides functions to trace the IBM/sarama package (https://github.com/IBM/sarama).
+package sarama // import "gopkg.in/DataDog/dd-trace-go.v1/contrib/IBM/sarama"
+
+import (
+	"math"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/telemetry"
+
+	"github.com/IBM/sarama"
+)
+
+const componentName = "IBM/sarama"
+
+func init() {
+	telemetry.LoadIntegration("IBM/sarama")
```

Comment:
We want to add a `tracer.MarkIntegrationImported("github.com/IBM/sarama.v1")` line here like we do in the current `sarama` package.